### DISCUSSION TITLE
Change one image on slide with multiple images

### DISCUSSION THREAD

**Discussion by Vennerstroem** [ShapeCrawler/ShapeCrawler#294] [Q&A]
Hi, 

How can i change one image in a slide without changing the others on the slide?

Currently i use SetImage(byte[] sourceBytes) in the IPicture shape, but when i set one image all other images change on the slide as well.

Some context i have multiple GroupShapes with dummy Images and Text and when i replace one of the dummy images all of them is replaced. The dummy images is essentially the same image, just copied.

**Comment by AdamShakhabov** [comment]
Hello.

@Vennerstroem, thank you for the bug report. I was able to reproduce this.

I hope to fix this soon.

**Reply by AdamShakhabov** [reply]
@Vennerstroem, the bug has been fixed in [**v0.33.0**](https://github.com/ShapeCrawler/ShapeCrawler/releases/tag/v0.33.0)
